Test is failing because of the extra newline.


```
  1) Jupyter converter
       should convert an exercise guide with an admonition block to ipynb:

      AssertionError: expected 'An admonition block may contain compl…' to equal 'An admonition block may contain compl…'
      + expected - actual

       - one
       - two
       - three
       
      -
       Another paragraph.
```

--- a/src/index.js
+++ b/src/index.js
@@ -175,7 +175,7 @@ class JupyterConverter {
         cells[0].source.unshift('\n')
       }
       if (cells.length > 1) {
-        cells[cells.length - 1].source.push('\n\n')
+        cells[cells.length - 1].source.push('\n')
       }
       return cells
     }

